src/commands/rover/NicknameGroupCommand.js

```diff
@@ -0,0 +1,41 @@
+const Command = require('../Command')
+
+module.exports =
+class NicknameGroupCommand extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'nicknamegroup',
+      properName: 'NicknameGroup',
+      aliases: ['rovernicknamegroup'],
+      description: "`<groupid>` Set the Group RoVer will use when setting users' nicknames.",
+
+      args: [
+        {
+          key: 'groupid',
+          label: 'groupid',
+          prompt: 'Nickname group',
+          type: 'string',
+          infinite: false
```

Comment:
Add `default: 0` and check if the group id is 0 before saving. If it's 0, set to undefined